host/sgx/linux/aep.S

```diff
@@ -34,3 +31,23 @@ OE_AEP:
     ud2
 
 .cfi_endproc
+
+
+//==============================================================================
+//
+// uint64_t OE_AEP_ADDRESS
+//
+//     The address of the ENCLU instruction is stored in this variable.
+//     If the OE_AEP function were to be used in code, the linker could create
+//     thunks that wrap the function. For example, when incremental linking is
+//     enabled, the linker on windows creates an entry in the ILT table for
+//     each function and uses that wherever the function is referenced.
+//     Thus OE_AEP would endup pointing to the thunk in the ILT which is not
```

Comment:
Typo: "end up", similarly in windows/aep.asm.

Reply:
Fixed.